contracts/test/integration/IntegrationTestMorphoCompoundPCVDeposit.t.sol

```diff
@@ -71,6 +76,8 @@ contract IntegrationTestMorphoCompoundPCVDeposit is DSTest {
 
         usdcDeposit.deposit();
         daiDeposit.deposit();
+
+        vm.roll(block.number + 3); /// fast forward 3 blocks so that profit is not negative
```

Comment:
why is that (and what are the "5 blocks to be in profit" you refer to in the `_withdraw` natspec ?

Reply:
Due to rounding down in solidity in favor of the protocol, that that is the breakeven amount of time.

Reply:
let me check and confirm the numbers

Reply:
ok, fixed these comments, after 1 block, all losses from precision loss on deposit are nullified and the deposit is in the green.